### PR TITLE
added PromiseWithoutRejectedCallbackFail

### PR DESCRIPTION
**Pull request by derekvanvliet**
I added a function that demonstrates issue #456. I'm finding that if I don't specify a rejected callback for the Parse.Config.get() promise's Then method, it never reaches my When method and the function times out. I found this worked in hosted parse but not in the open source version.

### CODE REVIEW

**Comment by flovilmart** [comment]
@derekvanvliet please rebase on master, we fixed the issue related with the failing test in #480

File: src/cloud/main.js [hunk 1]
```diff
@@ -102,3 +102,20 @@ Parse.Cloud.define('requiredParameterCheck', function(req, res) {
...
+
+Parse.Cloud.define("PromiseWithoutRejectedCallbackFail", function(request, response) {
+  var promises = [];
+	var result = {};
+
+	// fetch config
+	result.config = {};
+
+	promises.push(Parse.Config.get().then(function(config) {
+		result.config = config.attributes;
```
Comment (flovilmart) [inline src/cloud/main.js]: if you add `return Parse.Promise.as()` there, does it changes any thing?

Can you add a Unit test as well so the failure is properly handled and we ensure no regression in the future?

**Comment by drew-gross** [comment]
This test doesn't seem to fail in travis, it was a different test that failed. Close/open to trigger tests.

**Comment by flovilmart** [comment]
@drew-gross in that PR the function is actually never run...

**Comment by drew-gross** [comment]
Oops yeah missed that. @derekvanvliet can you add a test case that runs this cloud code?

**Comment by drew-gross** [comment]
Looks like there are merge conflicts, I guess you will have to resolve those before travis can run the tests.

**Comment by drew-gross** [comment]
Thanks! This will be very helpful as we attempt to fix the issue.

**Comment by flovilmart** [comment]
looks like the issue is more around Parse.Config.get() than the promises themselves.

**Comment by drew-gross** [comment]
Yeah that is true. @derekvanvliet Parse Config in Parse Server is currently in experimental stage, you can enable it with PARSE_EXPERIMENTAL_CONFIG_ENABLED=1 environment variable.

**Pull request closed without merging**